cmd/view/view_test.go

```diff
@@ -14,14 +14,13 @@ func TestView(t *testing.T) {
 	t.Run("NewViewCommand happy path", func(t *testing.T) {
 		client := azuremodels.NewMockClient()
 		modelSummary := &azuremodels.ModelSummary{
-			ID:           "test-id-1",
+			ID:           "openai/test-model-1",
```

Comment:
I thought I remembered talk recently of moving away from requiring the publisher name in model identifiers, something from @sgoedecke. Did that happen? Should we also have a test that just has the model name for its `ID` if so?